website/docusaurus.config.js

```diff
@@ -196,12 +196,12 @@ const isVersioningDisabled = !!process.env.DISABLE_VERSIONING || isI18nStaging;
           {
             tagName: 'link',
             rel: 'icon',
-            href: `${baseUrl}img/docusaurus.png`,
+            href: 'img/docusaurus.png',
```

Comment:
in general I'd rather always use a leading / and think we should someday force users to provide a leading / too.

those URLs are not relative URLs so let's make it explicit

Reply:
Debatable, for example I like the current implementation where base URL is automatically added to resource path, since end-user does not have to take care of it. Maybe we should document this better?